api-docs/paths/scim/team_details.json

```diff
@@ -0,0 +1,187 @@
+        "operationId": "Query an Individual Team",
+        "parameters": [
+            {
+                "name": "organization_slug",
+                "in": "path",
+                "required": true,
+                "schema": {
+                    "type": "string"
+                }
+            },
+            {
+                "name": "team_id",
+                "in": "path",
+                "required": true,
+                "description": "The id of the team you'd like to query.",
+                "schema": {
+                    "type": "integer"
+                }
+            }
+        ],
+        "responses": {
+            "200": {
+                "description": "Success",
+                "content": {
+                    "application/json": {
+                        "schema": {
+                            "$ref": "../../components/schemas/scim/group.json#/Group"
+                        },
+                        "example": {
+                            "schemas": [
+                                "urn:ietf:params:scim:schemas:core:2.0:Group"
+                            ],
+                            "id": "23232",
+                            "displayName": "test-scimv2",
+                            "members": [],
+                            "meta": {
+                                "resourceType": "Group"
+                            }
+                        }
+                    }
+                }
+            },
+            "401": {
+                "description": "Unauthorized"
+            },
+            "403": {
+                "description": "Forbidden"
+            }
+        },
+        "security": [
+            {
+                "auth_token": [
+                    "team: read"
+                ]
+            }
+        ]
+    },
+    "patch": {
+        "tags": [
+            "SCIM"
+        ],
+        "description": "Update a team's attributes with a SCIM Group PATCH Request. Valid Operations are:\n* Renaming a team: \n```json\n{\n    \"op\": \"replace\",\n    \"value\": {\n        \"id\": 23,\n        \"displayName\": \"newName\"\n    }\n}\n```\n* Adding a member to a team: \n```json\n{\n    \"op\": \"add\",\n    \"path\": \"members\",\n    \"value\": [\n        {\n            \"value\": 23,\n            \"display\": \"testexample@example.com\"\n        }\n    ]\n}\n```\n* Removing a member from a team: \n```json\n{\n    \"op\": \"remove\",\n    \"path\": \"members[value eq \\\"23\\\"]\"\n}\n``` \n*  Replacing an entire member set of a team: \n```json\n{\n    \"op\": \"replace\",\n    \"path\": \"members\",\n    \"value\": [\n        {\n            \"value\": 23,\n            \"display\": \"testexample2@sentry.io\"\n        },\n        {\n            \"value\": 24,\n            \"display\": \"testexample3@sentry.io\"\n        }\n    ]\n}\n```",
+        "operationId": "Update a Team's Attributes",
```

Comment:
Should `Team` be capitalized here? It isn't in the line above.

Reply:
"operationId" is in title format (what goes in the sidebar menu), so caps there.